linkerd/app/core/src/transport/mod.rs

```diff
@@ -1,5 +1,21 @@
 pub use linkerd_proxy_transport::*;
+use linkerd_stack::{ExtractParam, Param};
+pub use linkerd_transport_metrics as metrics;
+use std::sync::Arc;
 
 pub mod labels;
 
-pub type Metrics = metrics::Registry<labels::Key>;
+#[derive(Clone, Debug)]
+pub struct Metrics(metrics::Registry<labels::Key>);
```

Comment:
might be worth a comment explaining the relationship between this type and the `linkerd-transport-metrics` crate? it makes sense in the context of this PR and with the PR description, but i feel like it might be less obvious for someone looking at this code for the first time after this branch merges...

not a blocker though.